### ISSUE TITLE
customCommands enhancement

### ISSUE THREAD

**Issue by maku** [oliverschwendener/ueli#58]
You have to write the whole "prefix" of a customCommand plus the the first "variable" part to see anything  relevant in the search result:

e.g. I have a definition lke: 
 
```
"customCommands": [
    {
      "name": "Open with Visual Studio Code:",
      "executionArgument": "!code",
      "prefix": "openwithcode"
    }
  ],

```
I have to type 

![image](https://user-images.githubusercontent.com/64805/45761129-e8ca7c80-bc2b-11e8-9ff8-7c83eea73fef.png)

to see a result

Would it possible to present the command also in the search result? (After selecting in the search result and hit enter or tab the command should be presented in the input area to proceed with the variable part)

**Comment by oliverschwendener** [comment]
When would you expect your custom command to appear in the search results?

**Comment by oliverschwendener** [comment]
As a workaround for now: why don't you just set a shorter prefix?

**Comment by maku** [comment]
@oliverschwendener yeah would be max. a compromise solution. Problem is you would have to memorize the prefix. From experience I know that this does not work good... (especially when you have several...)

**Comment by oliverschwendener** [comment]
Allright. I will look into it as soon as possible.

**Comment by maku** [comment]
@oliverschwendener Great, thank you very much...
